threedi_schema/migrations/utils.py

```diff
@@ -0,0 +1,49 @@
+from typing import List
+
+import sqlalchemy as sa
+
+
+def drop_geo_table(op, table_name: str):
+    """
+
+    Safely drop table, taking into account geometry columns
+
+    Parameters:
+    op : object
+        An object representing the database operation.
+    table_name : str
+        The name of the table to be dropped.
+    """
+    connection = op.get_bind()
+    res = connection.execute(sa.text(f"SELECT DropGeoTable('{table_name}', 0);")).fetchall()[0][0]
```

Comment:
isn't this deprecated now we're running Spatialite 5?
DropTable should work now. See https://www.gaia-gis.it/fossil/libspatialite/wiki?name=Drop-RenameTable+and+RenameColumn

Reply:
Oops, I disabled the DropTable code to make the tests pass and forgot to restore that. Did so in b1577bacbcd6bbdd56ef409d39c11f37095f5910.